RedHatInsights/rhsm-subscriptions
# Update constraints on host table

https://issues.redhat.com/browse/SWATCH-715

* org_id: Add not-null constraint
* account_number: Remove not-null constraint

== Testing
1. Launch the application to trigger the liquibase update.
2. Ensure that `hosts.account_number` DOES NOT have the `not null` constraint.
3. Ensure that the `hosts.org_id`  HAS the `not null` constraint.
